worblehat-domain/src/test/java/de/codecentric/psd/worblehat/domain/StandardBookServiceTest.java

```diff
@@ -28,18 +27,29 @@
 
 	private static final DateTime NOW = DateTime.now();
 
-	private static final Book TEST_BOOK = new Book("title", "author", "edition", "isbn", 2016);
+	private static final Book A_BOOK = new Book("title", "author", "edition", "isbn", 2016);
+	private static final Book ANOTHER_BOOK = new Book("title2", "author2", "edition2", "isbn2", 2016);
+
+	private static final Book A_BORROWED_BOOK = new Book("title", "author", "edition", "isbn", 2016);
+	private static final Book ANOTHER_BORROWED_BOOK = new Book("title2", "author2", "edition2", "isbn2", 2016);
+	Set<Book> books;
+
+	{
+		A_BORROWED_BOOK.borrowNowByBorrower(BORROWER_EMAIL);
+		ANOTHER_BORROWED_BOOK.borrowNowByBorrower(BORROWER_EMAIL);
+	}
 
 	@Before
-	public void setup() throws Exception {
+	public void setup() {
+		books = new HashSet<>();
```

Comment:
As far as I can tell books will always be used with `Collections.addAll(books, A_BOOK, ANOTHER_BOOK);` in all tests, so we can move that code here. If you want to do it in a single line you can use `ImmutableSet.of(A_BOOK, ANOTHER_BOOK)` from google guava.

Reply:
No, sometimes also borrowed books are added to that collection. And sometimes a mix of borrowed and unborrowed books.